pdsview/pdsview.py

```diff
@@ -9,7 +9,7 @@
 # Please see the file LICENSE.txt for details.
 #
 import sys
-import logging
+# import logging
```

Comment:
We should figure out how to use logging properly.  I bet Ginga has an example of how to do so.